protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -525,6 +551,11 @@ func schemaOfField(f *descriptor.Field, reg *descriptor.Registry, refs refMap) o
 		if wktSchema, ok := wktSchemas[fd.GetTypeName()]; ok {
 			core = wktSchema
 
+			// if wfkSchema is a reference to another known type, add it to refs to be processed later
+			if core.Ref != "" {
+				refs[fd.GetTypeName()] = struct{}{}
+			}
+
 			if fd.GetTypeName() == ".google.protobuf.Empty" {
 				props = &openapiSchemaObjectProperties{}
 			}
```

Comment:
This can be tidied up now, and `props` for `.google.protobuf.Empty` be defined on the `wktSchemas` map. Can address this later (I need to log off for a few hours)